src/Microsoft.TestPlatform.ShimLayer/StringExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I have named the containing folder `Microsoft.TestPlatform.ShimLayer`, let me know if you prefer a simple and maybe more generic name for such kind of shared file.